[Homework #5] - CSS-Advanced by Pavel Didenko

Win4esterD opened the pull request:
@ilya-ischenko

Fullfilled all the remarks left under the previous task and implemented all the features needed to pass 5th task.

Implemented hover effects for all clickable elements except "scroll to top" button, because this button is only to be used on mobile devices, so there is no way users can call it.

ilya-ischenko reviewed (commented):
Hi! Actually looks good!

1. https://ibb.co/YfGDG9z (754px) - portfolio and skills button is not aligned or container is too big for them
2. https://ibb.co/SvzGnwy (756px) - the same for portfolio and skills button. Need to add paddings left/right for content in order to have good ux (content should not be placed at the edge of the screen)
3. https://ibb.co/YBfsCqS - you can also maintain smaller devices by adding padding left/right for your website.
4. Seems to be redundant icon - homeworks/pavel-didenko/05-css-advanced/assets/img/icons/scroll-to-top.png

Win4esterD commented:
1) Aligned buttons using "justify-content: center" and modified their width on different resolutions.
2) Didn't understand the trick with paddings, seems like didn't work for me, but at the same time I used "max-width: 90%" for the container of portfolios. Looks like it did the trick well, now the portfolios have idents on small screens.
3) Miss the point, doesn't work for me.
4) Removed extra icon.

ilya-ischenko reviewed (approved):
Well done!
Approved.
